lib/monitoring/aws-secretsmanager/SecretsManagerMetricFactory.ts

```diff
@@ -0,0 +1,37 @@
+import { Duration } from "aws-cdk-lib";
+import { MetricFactory, MetricStatistic } from "../../common";
+
+const DEFAULT_METRIC_PERIOD = Duration.hours(1);
+
+export class SecretsManagerMetricFactory {
+  static readonly Namespace = "AWS/SecretsManager";
```

Comment:
We've typically had these outside of the factory class for other similar things (i.e. at the same level as `DEFAULT_METRIC_PERIOD` here).